pkg/upgrade/upgrade.go

```diff
@@ -200,17 +200,21 @@ func (u *Upgrader) nextUpdate() time.Duration {
 func (u *Upgrader) remote() (remote hub.TagLink, err error) {
 	mode := u.boot.RunMode()
 	// find all taglinks that matches the same run mode (ex: development)
+	envName := mode.String()
+	if u.boot.Version().Major == 4 {
+		envName = fmt.Sprintf("%s-light", envName)
+	}
```

Comment:
Since this is a fork and the entire fork knows that this is version 4. Hence you can always assume the envName to be "<env>-light" without even the need to check for the major version